jobs/buildhub/configure_markus.py

```diff
@@ -0,0 +1,41 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import markus
+from decouple import config
+
+
+_configured = False
+
+
+def get_metrics(namespace):
+    global _configured
+    if not _configured:
+        STATSD_HOST = config('STATSD_HOST', 'localhost')
+        STATSD_PORT = config('STATSD_PORT', default=8125)
+        STATSD_NAMESPACE = config('STATSD_NAMESPACE', default='')
+
+        # For more options see
+        # http://markus.readthedocs.io/en/latest/usage.html#markus-configure
+        markus.configure([
+            # Deliberately commented out. Uncomment if you want to see
+            # what's sent to markus in local development.
+            # {
+            #     'class': 'markus.backends.logging.LoggingMetrics',
+            #     'options': {
+            #         'logger_name': 'metrics'
+            #     }
+            # },
```

Comment:
we'd rather add a config variable to include/exclude this block instead of using code un/commenting

Reply:
yeah, removed now. There's a URL in the comment to the documentation for markus.